src/main/resources/version

```diff
@@ -0,0 +1 @@
+${version}
```

Comment:
Missing newline

Reply:
I just read the file as a plain string. A new line would be removed by discord anyway 🤔